source/distributed/fully_distributed_tria.cc

```diff
@@ -561,13 +562,13 @@ namespace parallel
         AssertThrowMPI(ierr);
 
         // Write buffers to file.
-        ierr = MPI_File_write_at(fh,
-                                 mpisize * sizeof(unsigned int) +
-                                   offset, // global position in file
-                                 buffer.data(),
-                                 buffer.size(), // local buffer
-                                 MPI_CHAR,
-                                 MPI_STATUS_IGNORE);
+        ierr = dealii::Utilities::MPI::LargeCount::MPI_File_write_at_c(
+          fh,
+          mpisize * sizeof(unsigned int) + offset, // global position in file
```

Comment:
Can you compute this position before and store it in a 64bit int?

Reply:
like this?

```
MPI_Offset offset = 0;

MPI_Offset global_position = mpisize * sizeof(unsigned int) + offset;
```